src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java

```diff
@@ -37,6 +37,10 @@
  */
 public final class MinecraftVersion implements Comparable<MinecraftVersion>, Serializable {
 
+    /**
+     * Version 1.20.2 - the update that added the configuration protocol phase.
+     */
+    public static final MinecraftVersion CONFIG_PHASE_PROTOCOL_UPDATE = new MinecraftVersion("1.20.2");
```

Comment:
If someone has a better name suggestion please let me know. I figured it would be nice to have this to allow libs to check the version due to the rather big protocol changes.

Reply:
maybe BANDWIDTH_UPDATE or similar?